packages/backend/src/app/authentication/authentication.service.ts

```diff
@@ -37,7 +37,7 @@ export const authenticationService = {
                     projectId: project.id,
                 },
             })
-            user.password = undefined
+            user.password = ''
```

Comment:
Please add todo to make it undefined so api response doesn't return empty string

Reply:
fixed in f9e85866